Source/WebKit/UIProcess/Extensions/Cocoa/WebExtensionControllerCocoa.mm

```diff
@@ -259,6 +260,16 @@
     return extensions;
 }
 
+#if PLATFORM(MAC)
+void WebExtensionController::addItemsToContextMenu(WebPageProxy& page, const ContextMenuContextData& contextData, NSMenu *menu)
+{
+    [menu addItem:NSMenuItem.separatorItem];
```

Comment:
Will there always be menu items in the menu above this?

Reply:
WebKit culls useless or redundant separators before showing the menu, so this is safe.